src/components/application_manager/src/commands/mobile/show_constant_tbt_request.cc

```diff
@@ -181,6 +181,7 @@ void ShowConstantTBTRequest::on_event(const event_engine::Event& event) {
   switch (event.id()) {
     case hmi_apis::FunctionID::Navigation_ShowConstantTBT: {
       LOG4CXX_INFO(logger_, "Received Navigation_ShowConstantTBT event");
+      const char* return_info = NULL;
```

Comment:
@OHerasym,
- assignment to `const char* return_info` looks wring. remove const at least
- why you need brackets around the `message[strings::msg_params][hmi_response::message].asString()`?